lib/ui/login_screen.dart

```diff
@@ -1,96 +1,114 @@
+                          // softWrap: true,
+                          textAlign: TextAlign.center,
+                          style: GoogleFonts.montserrat(
+                              color: Styles.buttonColor,
+                              fontSize:
+                                   aspectRatio*
+                                     75,
+                              fontWeight: FontWeight.w400),
+                        ),
+                      ),
+                    ],
+                  ),
+                ),
+              ],
             ),
             SizedBox(
               height: deviceHeight * 0.052681,
             ),
             Expanded(
               child: Align(
                 alignment: Alignment.bottomCenter,
-                child: InkWell(
-                  onTap: () {
+                child: RaisedButton(
+                  elevation: 7,
+                  shape: RoundedRectangleBorder(
+                    borderRadius: BorderRadius.circular(deviceHeight * 0.03634),
+                    // side: BorderSide(color: Colors.red)
+                  ),
+                  color: Styles.backgroundColor,
+                  onPressed: () {
                     var googleSignMe =
                         GoogleSignMeIn(whatToDo: 1, context: context);
                     googleSignMe.check();
                   },
                   child: Container(
-                    height: deviceHeight * 0.0675852,
-                    width: deviceWidth * 0.8,
-                    decoration: BoxDecoration(
-                      gradient: LinearGradient(
-                        begin: Alignment.bottomLeft,
-                        colors: colorList,
-                        end: Alignment.topRight,
-                      ),
-                      borderRadius:
-                          BorderRadius.circular(deviceHeight * 0.02634),
-                    ),
+                    height: deviceHeight * 0.07675852,
+                    width: deviceWidth * 0.76,
                     child: Row(
                       children: [
                         Container(
                           padding: EdgeInsets.only(left: deviceWidth * 0.011),
                           child: Image.asset(
                             "assets/icons/google_bg.png",
                             fit: BoxFit.fitHeight,
+                            scale: 1.5,
                           ),
                         ),
-                        Expanded(
+                        Flexible(
                           child: Text(
-                            " Sign In with Google",
-                            style: GoogleFonts.roboto(
-                              color: Colors.white,
-                              fontSize: deviceHeight * 0.02711292,
-                            ),
+                            " Log in with Google",
+                            style: GoogleFonts.montserrat(
+                                fontSize: aspectRatio * 45,
```

Comment:
Keep the font sizes fixed irrespective of the screen size. It gets messy when we test applications on devices having different aspect ratios. However, it's okay to use this method for other visual components which need to be dynamic.

Define different TextStyles in `styles.dart` with all the necessary parameters like font weight, font size, font colour etc.
You should have a fixed set of consistent font styles defined in the file which could be used directly in the UI components.

Reply:
I create an issue for extra info of text styles in styles.dart file